app/models/article.rb

```diff
@@ -1,9 +1,22 @@
 class Article < ApplicationRecord
-  validates_presence_of :title, :teaser, :body, :category
-  validates_inclusion_of :premium, in: [false, true]
-  scope :most_recent, -> { order(updated_at: :desc) }
+  validates_inclusion_of :backyard, in: [false, true]
```

Comment:
How does this work?

Reply:
Validates that the backyard attribute is either true or false

Reply:
It checks whether the attribute for backyard is true or false, this works kind of in the same way we have with the premium attribute

Reply:
Thanks